.circleci/config.yml

```diff
@@ -129,13 +141,27 @@ jobs:
       - attach_workspace:
           at: /tmp/workspace
       - checkout
+      - restore_cache:
+          keys:
+            # Use own cache for front-end builds, since for some reason it can't be unpacked for parts of the
+            # pipeline
+            - v1-yarn-build-{{ checksum "yarn.lock" }}
+            # Used if checksum fails
+            - v1-yarn-build-
       - run:
           name: Increase node memory
           command: |
             export NODE_OPTIONS=--max_old_space_size=4096
       - run:
           name: CI job started
           command: ./scripts/ci-job-started.sh
+      - run:
+          name: Install dependencies
+          command: yarn install --pure-lockfile
```

Comment:
Using `yarn install --pure-lockfile --no-progress` I think is faster, at least it is for me locally.